kotlin/configure-subnetwork-trace/src/main/res/layout/activity_main.xml

```diff
@@ -6,175 +6,213 @@
+                app:layout_constraintTop_toBottomOf="@+id/sourceBackgroundView">
+
+                <android.widget.Spinner
+                    android:id="@+id/operatorSpinner"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content" />
+            </android.widget.RelativeLayout>
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/layout1"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toBottomOf="@+id/operatorBackgroundView">
+
+                <android.widget.RelativeLayout
+                    android:id="@+id/valuesBackgroundView"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginEnd="16dp"
+                    android:background="@drawable/border"
+                    android:padding="4dp"
+                    android:visibility="visible"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent">
+
+                    <android.widget.Spinner
+                        android:id="@+id/valuesSpinner"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content" />
+                </android.widget.RelativeLayout>
+
+                <android.widget.EditText
+                    android:id="@+id/valuesEditText"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="16dp"
+                    android:layout_marginTop="8dp"
+                    android:layout_marginEnd="16dp"
+                    android:hint="Value"
+                    android:inputType="number"
+                    android:padding="10dp"
+                    android:visibility="visible"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/valuesBackgroundView" />
+
+                <android.widget.ToggleButton
+                    android:id="@+id/valueBooleanButton"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="8dp"
+                    android:textOff="@string/falseText"
+                    android:textOn="@string/trueText"
+                    android:visibility="visible"
```

Comment:
setting this to gone will keep the button from popping up during onCreate, then disappearing